src/main/java/TaskManager.java

```diff
@@ -0,0 +1,35 @@
+public class TaskManager {
+    public static final int MAXIMUM_TASK = 100;
+    private Task[] tasks = new Task[MAXIMUM_TASK];
+    private int numOfTasks = 0;
+
+    public void addTask(String command) {
+        System.out.println("------------------------------------");
+        System.out.println("added: "+ command);
+        System.out.println("You added a new task, remember to finish it on time!");
+        System.out.println("------------------------------------");
+        System.out.println();
+        Task task = new Task();
+        task.name = command;
+        task.isDone = false;
+        tasks[numOfTasks++] = task;
+    }
+
+    public void listTask() {
+        System.out.println("------------------------------------");
+        System.out.println("Here is the list of all tasks you have:");
+        for(int i = 0; i < numOfTasks; i++) {
+            System.out.println(String.format("%d.[%c] %s", i+1,(tasks[i].isDone? '✓':'✗'), tasks[i].name));
```

Comment:
You may want to consider seperating this statement into a few more lines to make the code more readable. For example, you may want to first create variables for each item in the print function:
String isDone = tasks[i].isDone? '✓':'✗';